docs/user-guide/configuring.md

```diff
@@ -1070,12 +1070,26 @@ Of particular note is that like `.gitignore` files, all paths used as patterns f
 
 Please see `.gitignore`'s specification for further examples of valid syntax.
 
-In addition to any patterns in a `.eslintignore` file, ESLint ignores files in `/**/node_modules/*` by default. It can still be added using `!`.
+In addition to any patterns in the `.eslintignore` file, ESLint always follows a couple implicit ignore rules even if the `--no-ignore` flag is passed. The implicit rules are as follows:
```

Comment:
Can we also mention `--ignore-pattern` here?

Reply:
The sample `.eslintignore` file I had near the bottom of my changes lost some meaning after a rebase, I think. I converted it into an additional bullet point mentioning that `.eslintignore` and `--ignore-pattern` are prioritized above implicit rules.